source/NVDAObjects/IAccessible/sysListView32.py

```diff
@@ -274,11 +274,15 @@ def _getColumnOrderArrayRawOutProc(self, columnCount: int) -> Optional[ctypes.Ar
 	def _getColumnOrderArrayRaw(self, columnCount: int) -> Optional[ctypes.Array]:
 		"""Retrieves an array of column indexes for a given list.
 		The indexes are placed in order in which columns are displayed on screen from left to right.
-		Note that when columns are reordered the indexes remain  the same - only their ordder differs.
+		Note that when columns are reordered the indexes remain the same - only their order differs.
 		"""
-		if not self.appModule.helperLocalBindingHandle:

```

Comment:
I'm worried about the removal of this line. WE definitely shouldn't try in process when self.appModule.helperLocalBindingHandle is None.

Reply:
this should be fixed now, and the fix still works